region/crateria/east/The Moat.json

```diff
@@ -1137,11 +1135,6 @@
     }
   ],
   "notables": [
-    {
-      "id": 1,
-      "name": "SpringBall Bounce",
-      "note": "From an open doorway or adjacent room, run, jump, lateral midair morph on the way down, then bounce off the pedestal to get to the other side."
-    },
```

Comment:
This one is very interesting and has vanilla uses, but it's not really much more notable over any other spring ball bounce. Instead of being notable, we switched the tech video be this (instead of bubble mountain)